pacman/utilities/utility_objs/resource_tracker.py

```diff
@@ -100,10 +100,18 @@ class ResourceTracker(object):
         "_real_chips_with_n_cores_available",
 
         # the number of virtual chips with the n cores currently available
-        "_virtual_chips_with_n_cores_available"
+        "_virtual_chips_with_n_cores_available",
+
+        # tracker of vertex to chip location
+        '_vertex_to_chip_map',
 
     ]
 
+    ALLOCATION_SDRAM_ERROR = (
+        "allocating of {} SDRAM on chip {}:{} has failed as there is "
```

Comment:
Should say "Allocating of {} bytes of SDRAM on chip {}:{} has failed as there are " 
                   "only {} bytes of SDRAM available on the chip .... "

Reply:
fixed